guides/common/modules/ref_example-playbooks-based-on-modules-from-project-ansible-collection.adoc

```diff
@@ -0,0 +1,153 @@
+[id="example-playbooks-based-on-modules-from-{project-context}-ansible-collection"]
+= Example playbooks based on modules from {Project} Ansible Collection
+
+All playbooks based on modules from {Project} Ansible Collection must include parameters detailing how to connect to the {Project} API.
+The following examples use Ansible vault and module defaults group to provide these parameters, and they authenticate using a username and password.
+For more information, see xref:creating-a-playbook-with-modules-from-{project-context}-ansible-collection[].
+
+.Example Ansible playbook: Enable repositories and create a content view
+====
+This example playbook uses the following modules:
+
+* `{ansible-namespace}.repository_set`
+* `{ansible-namespace}.content_view`
+
+The playbook ensures repositories are enabled and a content view that contains these repositories exists.
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+- name: Ensure RHEL 10 repositories are enabled and a content view exists
+  hosts: localhost
+  vars_files:
+    - _My_Vault.yml_
+  module_defaults:
+    group/{ansible-module-defaults-group}:
+      username: "{{ _My_Username_ }}"
+      password: "{{ _My_Password_ }}"
+      server_url: "{{ _My_Server_URL_ }}"
+  tasks:
+    - name: Ensure RHEL 10 repositories are enabled
+      {ansible-namespace}.repository_set:
+        name: "Red Hat Enterprise Linux 10 for x86_64 - BaseOS (RPMs)"
```

Comment:
I'd use RHEL9, just for the sake of it already being released :)

Reply:
Replaced